resolwe_bio/processes/chip_seq/macs2_rose2_batch.yml

```diff
@@ -17,7 +17,7 @@
           - case
         required: True
   data_name: ChIP-Seq Workflow Batch
-  version: 1.0.3
+  version: 1.1.0
```

Comment:
For this file, only the order of option changes and `promoter` hidden status changes. Does this warrant a middle version bump?